<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1160</width>
    <height>852</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>1159</width>
      <height>832</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QWidget" name="icon_only_widget" native="true">
       <property name="minimumSize">
        <size>
         <width>71</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>71</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget{
	background-color: rgb(255, 170, 0);
}

QPushButton:checked { 
	background-color:white;
	border-radius: 3px;
}</string>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>61</width>
          <height>42</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="resources.qrc">:/Icons/petoi_icon.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>4</x>
          <y>60</y>
          <width>114</width>
          <height>381</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QPushButton" name="dashboard_1">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/Icons/dashboardsmall1.png</normaloff>
             <normalon>:/Icons/dashboardsmall2.png</normalon>:/Icons/dashboardsmall1.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="about_1">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/Icons/about_icon.png</normaloff>:/Icons/about_icon.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>100</width>
             <height>16</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="voice_1">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/Icons/voice_icon.png</normaloff>:/Icons/voice_icon.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>100</width>
             <height>20</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>750</y>
          <width>114</width>
          <height>81</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <widget class="QPushButton" name="settings_1">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/Icons/settingssmall1.png</normaloff>
             <normalon>:/Icons/settingssmall2.png</normalon>:/Icons/settingssmall1.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>100</width>
             <height>20</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="signout_1">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/Icons/signoutsmall1.png</normaloff>
             <normalon>:/Icons/signoutsmall2.png</normalon>:/Icons/signoutsmall1.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>100</width>
             <height>16</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="icon_text_widget" native="true">
       <property name="minimumSize">
        <size>
         <width>241</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>241</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget{
	background-color: rgb(255, 170, 0);
	color: white;
}

QPushButton { 
	height: 30px;
	border: none;
}</string>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>750</y>
          <width>221</width>
          <height>84</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QPushButton" name="settings_2">
           <property name="styleSheet">
            <string notr="true">QPushButton { 
	padding-left: -60px;
}

QPushButton:checked { 
	background-color: white;
	border-radius:3px;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/Icons/settings2.png</normaloff>
             <normalon>:/Icons/settings1.png</normalon>:/Icons/settings2.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>100</width>
             <height>16</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="signout_2">
           <property name="styleSheet">
            <string notr="true">QPushButton { 
	padding-left: -60px;
}

QPushButton:checked { 
	background-color: white;
	border-radius:3px;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/Icons/signout2.png</normaloff>
             <normalon>:/Icons/signout1.png</normalon>:/Icons/signout2.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>100</width>
             <height>16</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>221</width>
          <height>42</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="resources.qrc">:/Icons/petoi_icon.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <pointsize>20</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Voice Control</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QFrame" name="voice">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>320</y>
          <width>206</width>
          <height>261</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <widget class="QPushButton" name="voice_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>204</width>
           <height>62</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton { 
	padding-left:0px
}

QPushButton:hover{ 
	color: #12B298
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/Icons/voice_icon.png</normaloff>:/Icons/voice_icon.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>200</width>
           <height>60</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QFrame" name="voice_dropdown">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>70</y>
           <width>211</width>
           <height>104</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="voice_dashboard">
            <property name="text">
             <string>Voice Dashboard</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="voice_playground">
            <property name="text">
             <string>Voice Playground</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="voice_settings">
            <property name="text">
             <string>Voice Settings</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QPushButton" name="about_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>220</y>
          <width>201</width>
          <height>50</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { 
	padding-left: -66px;
}

QPushButton:checked { 
	background-color: white;
	border-radius:3px;
}</string>
        </property>
        <property name="text">
         <string>About</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/Icons/about_icon.png</normaloff>
          <normalon>:/Icons/about_icon.png</normalon>:/Icons/about_icon.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>200</width>
          <height>45</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="dashboard_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>130</y>
          <width>211</width>
          <height>30</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { 
	padding-left: -60px;
}

QPushButton:checked { 
	background-color: white;
	border-radius:3px;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/Icons/dashboard2.png</normaloff>
          <normalon>:/Icons/dashboard1.png</normalon>:/Icons/dashboard2.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>100</width>
          <height>16</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <item>
        <widget class="QWidget" name="header_widget" native="true">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="styleSheet">
               <string notr="true">border: none;</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/Icons/menu.png</normaloff>:/Icons/menu.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>29</width>
                <height>35</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_11">
              <item>
               <widget class="QLabel" name="label">
                <property name="font">
                 <font>
                  <pointsize>15</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Hello, [Person's Name]</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_5">
                <property name="styleSheet">
                 <string notr="true">color: rgb(131, 131, 131);</string>
                </property>
                <property name="text">
                 <string>[Pet's Name] Home Portal</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>315</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="resources.qrc">:/Icons/petoi_icon.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="main_screen_widget" native="true">
         <property name="minimumSize">
          <size>
           <width>831</width>
           <height>751</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>831</width>
           <height>751</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(230, 230, 230);</string>
         </property>
         <widget class="QStackedWidget" name="stackedWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>811</width>
            <height>731</height>
           </rect>
          </property>
          <widget class="QWidget" name="page">
           <widget class="QLabel" name="label_7">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>0</y>
              <width>391</width>
              <height>61</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>25</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Dashboard</string>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="page_2">
           <widget class="QLabel" name="label_8">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>0</y>
              <width>391</width>
              <height>61</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>25</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>About</string>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="page_3">
           <widget class="QLabel" name="label_9">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>0</y>
              <width>391</width>
              <height>61</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>25</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Voice Dashboard</string>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="page_4">
           <widget class="QLabel" name="label_10">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>0</y>
              <width>391</width>
              <height>61</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>25</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Voice Playground</string>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="page_5">
           <widget class="QLabel" name="label_11">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>0</y>
              <width>391</width>
              <height>61</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>25</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Voice Settings</string>
            </property>
           </widget>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>settings_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>settings_1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>791</y>
    </hint>
    <hint type="destinationlabel">
     <x>70</x>
     <y>792</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>signout_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>signout_1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>833</y>
    </hint>
    <hint type="destinationlabel">
     <x>70</x>
     <y>835</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>signout_1</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>832</y>
    </hint>
    <hint type="destinationlabel">
     <x>91</x>
     <y>905</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>signout_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>837</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>928</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>icon_text_widget</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>358</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>icon_only_widget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>54</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dashboard_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>dashboard_1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>62</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>voice_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>voice_dropdown</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>366</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>490</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>voice_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>voice_1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>73</x>
     <y>340</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>about_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>about_1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>57</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
